rasa/shared/utils/cli.py

```diff
@@ -6,15 +6,17 @@
 
 def print_color(*args: Any, color: Text) -> None:
     output = rasa.shared.utils.io.wrap_with_color(*args, color=color)
-    try:
+    stream = sys.stdout
+    if sys.platform == "win32":
         # colorama is used to fix a regression where colors can not be printed on
         # windows. https://github.com/RasaHQ/rasa/issues/7053
         from colorama import AnsiToWin32
 
         stream = AnsiToWin32(sys.stdout).stream
+    try:
         print(output, file=stream)
```

Comment:
Do we actually need the `stream` variable here for all platforms? In the original code it seems to be valid only for Windows 🤔

Reply:
By default `print` gets called with  `end='\n', file=sys.stdout, flush=False` I just made it explicit to not have the two `print` statements as before